### PR TITLE
[NEW] Adds preference to one-click-to-direct-message and basic functionality

### LINKED ISSUE #7436: Create an option to go straight to direct message when clicking on a user's name
Right now, when you click on a user's name it goes to the profile page. Many users will prefer to go directly to chat with that user.

### PR DESCRIPTION
**Pull request by gdelavald**
@RocketChat/core 

Closes #7436

This feature adds a preference to users who would prefer to skip opening the flex tab with personal information when clicking on a username/card and goes straight into direct message mode.

### CODE REVIEW

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 1]
```diff
@@ -330,6 +330,8 @@
   "Clear_all_unreads_question": "Clear all unreads?",
   "Click_here": "Click here",
   "Click_here_for_more_info": "Click here for more info",
+  "Click_Direct_Message": "Click to Create Direct Message",
+  "Click_Direct_Message_Description": "Skip opening profile tab and goes straight to conversation",
```
Comment (marceloschmidt) [inline packages/rocketchat-i18n/i18n/en.i18n.json]: Either `Skips opening ... and goes ...` or `Skip opening ... and go ...` -- I prefer the former.

Comment (antgel) [inline packages/rocketchat-i18n/i18n/en.i18n.json]: "Skip opening profile tab, instead go straight to conversation" :D

**Comment by gdelavald** [comment]
@sampaiodiego what do you think should be the behavior when clicking on yourself or for users that don't have direct channel permission?

**Comment by sampaiodiego** [comment]
@gdelavald when clicking on yourself or for users that don't have direct channel permission I think we should show the profile panel, like the old/default behavior.

**Comment by rodrigok** [comment]
IMHO this should be an admin option instead of per user. The use case right now is to change the chat behavior to all users. We can add a sub setting in the future to change the default behavior per user.

**Pull request merged**